tasks/src/depth-estimation/data.ts

```diff
@@ -0,0 +1,37 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [],
+	demo: {
+		inputs: [
+			{
+				filename: "depth-estimation-input.jpg",
+				type:     "img",
+			},
+		],
+		outputs: [
+			{
+				filename: "depth-estimation-output.png",
+				type:     "img",
+			},
+		],
+	},
+	metrics: [],
```

Comment:
I think you can write something like Contribute a metric so people will be contributing it. (same applies for dataset, you can link this dataset if there's none huggingface-projects/contribute-a-dataset)

Reply:
Are there any examples I can refer to?